src/devices/Mlx90614/Register.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Mlx90614
+{
+    internal enum Register
+    {
+        MLX_AMBIENT_TEMP = 0x06,
+        MLX_OBJECT1_TEMP = 0x07,
+        MLX_OBJECT2_TEMP = 0x08
```

Comment:
what is MLX_OBJECT2_TEMP? It doesn't seem to be used anywhere?

Reply:
In the application guide, the specific version of the sensors read the 0x08 register effectively, while the 0x07 register is universal. The functions of these two registers are the same. I will remove 0x08